main.go

```diff
@@ -0,0 +1,18 @@
+/*
```

Comment:
can this be removed?

Reply:
The file? Yes and no :)

Yes, we can remove it and everything will work fine.

Things will not work fine IF/WHEN we try to create a new scaffold for either a resource or a controller using `kubebuilder`. `kubebuilder` will halt the generation of files when it doesn't find `main.go`. Well, we don't do this everyday (generate new resources/controllers) so (again, if we go in this direction) we COULD just remember to add a `main.go` file while doing that and remove it again.